src/main/java/org/example/utility/UtilityMethods.java

```diff
@@ -97,4 +97,18 @@ public static int countOdd(int[] array) {
         }
         return count;
     }
+
+    public static int findIndex(int[] array, int number) {
+        checkNotNull(array, "Your array is null");
+        int result = 0;
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == number) {
+                result = i;
+                break;
```

Comment:
I think your implementation can be simplified. Instead of breaking a loop you can return the index `i` immediately. So you don't need the `result` variable